packages/shared/src/components/profile/ProfileButton.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactElement, useContext } from 'react';
+import dynamic from 'next/dynamic';
+import { LazyImage } from '../LazyImage';
+import AuthContext from '../../contexts/AuthContext';
+import { getTooltipProps } from '../../lib/tooltip';
+import useProfileMenu from '../../hooks/useProfileMenu';
+
+const ProfileMenu = dynamic(
+  () => import(/* webpackChunkName: "profileMenu" */ '../ProfileMenu'),
+);
+
+export interface ProfileButtonProps {
+  onShowDndClick?: () => unknown;
+}
+
+export default function ProfileButton({
+  onShowDndClick,
+}: ProfileButtonProps): ReactElement {
+  const { user } = useContext(AuthContext);
+  const profileContextMenuWidth = 10;
```

Comment:
This can be taken out of the component. You can store it as a global variable in this file.